src/utils/pgtype-binary/Cargo.toml

```
@@ -0,0 +1,19 @@
+[package]
+name = "pgtype-binary"
+version = "0.1.0"
+edition = "2021"
+description = "postgresql type binary format."
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+bytes = "1"
+chrono = { version = "0.4", default-features = false, features = ["clock", "std"] }
+postgres-types = { version = "0.X.X", features = ["derive","with-chrono-0_4"] }
```

```suggestion
postgres-types = { version = "0.2", features = ["derive","with-chrono-0_4"] }
```
"0.X.X" Is that working?? I believe it should be replaced by some numbers.